mkdocs.yml

```diff
@@ -163,11 +164,6 @@ markdown_extensions:
           - site/overrides/.icons
 
 plugins:
-    - search
-    - redirects:
-        redirect_maps:
-            'docs/installing.md': 'docs/getting-started/install.md'
-            'docs/getting-started.md': 'docs/getting-started/first-app-tutorial.md'
```

Comment:
because i don't think this is applicable any more since `docs/getting-started/install.md` is not a valid address as well. Maybe i'll just make them valid again

Reply:
icic... seems like we'd still need `-search` though